### PR TITLE
Feature/hidden icons in mobile

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
https://trello.com/c/qYKTvz5G/236-feature-delegatestransparencyreporting-v1

# Actions
Add background page

Add breadcrumbs customized](https://github.com/makerdao-ses/ecosystem-

Add hidden icon and customized breadcrumbs component

### CODE REVIEW

File: src/stories/components/breadcrumbs/customize-breadcrumbs/customize-breadcrums.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { siteRoutes } from '@ses/config/routes';
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import CustomizeBreadCrumbs from './customize-breadcrums';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/General/CustomizeBreadCrumbs',
+  component: CustomizeBreadCrumbs,
```
Comment (YasielCabrera) [inline src/stories/components/breadcrumbs/customize-breadcrumbs/customize-breadcrums.stories.tsx:8]: This component looks different in several breakpoints, you should add the chromatic viewports to detect those changes

File: src/stories/components/breadcrumbs/customize-breadcrumbs/customize-breadcrums.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+interface Props {
+  items: {
+    label: string | JSX.Element;
+    url: string;
+  }[];
+  isLight: boolean;
+}
+
+const CustomizeBreadCrumbs = ({ items, isLight }: Props) => (
```
Comment (YasielCabrera) [inline src/stories/components/breadcrumbs/customize-breadcrumbs/customize-breadcrums.tsx:13]: This should be `CustomBreadCrumbs` right? "Customize" is an action

File: src/stories/components/delegate-summary/delegate-summary.stories.tsx [hunk 3]
```diff
@@ -47,41 +58,41 @@ LightMode.parameters = {
...
         options: {
           componentStyle: {
-            width: 343,
+            width: 375,
           },
           style: {
-            top: -18,
-            left: -22,
+            top: -36,
+            left: -40,
```
Comment (YasielCabrera) [inline src/stories/components/delegate-summary/delegate-summary.stories.tsx:68]: Please, check the integration, the component don't fully match with the design

File: src/stories/components/delegate-summary/delegate-summary.tsx [hunk 4]
```diff
@@ -11,79 +13,152 @@ import type { LinkModel } from '../cu-table-column-links/cu-table-column-links';
...
   const isUp1280 = useMediaQuery(lightTheme.breakpoints.up('table_834'));
+  const isMobile = useMediaQuery(lightTheme.breakpoints.between('table_375', 'table_834'));
+  const [showIcons, setShowIcons] = useState(true);
+  const [positionScroll, setPositionScroll] = useState(0);
+
+  const scrollMargin = 20;
+
+  const handleScroll = () => {
+    const position = window.pageYOffset || document.documentElement.scrollTop;
+    console.log(position);
```
Comment (YasielCabrera) [inline src/stories/components/delegate-summary/delegate-summary.tsx:33]: Is this necessary?

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/components/breadcrumbs/customize-breadcrumbs/customize-breadcrums.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/components/breadcrumbs/customize-breadcrumbs/customize-breadcrums.tsx:13]: you are right, done

File: src/stories/components/delegate-summary/delegate-summary.tsx (hunk 4 above)
Comment (alejandrocabriales) [reply src/stories/components/delegate-summary/delegate-summary.tsx:33]: no, delete it

File: src/stories/components/breadcrumbs/customize-breadcrumbs/customize-breadcrums.stories.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/components/breadcrumbs/customize-breadcrumbs/customize-breadcrums.stories.tsx:8]: ok, done

File: src/stories/components/delegate-summary/delegate-summary.stories.tsx (hunk 3 above)
Comment (alejandrocabriales) [reply src/stories/components/delegate-summary/delegate-summary.stories.tsx:68]: already for the view of 375, 834, and 1994, in the other view the component don't change

**Review by YasielCabrera — APPROVED** [review]
LGTM

**Pull request merged**